features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/package.json

```diff
@@ -66,6 +66,7 @@
         "@babel/preset-env": "^7.4.4",
         "@babel/preset-react": "^7.0.0",
         "@babel/register": "^7.4.4",
+        "@material-ui/lab": "^4.0.0-alpha.29",
```

Comment:
Any particular reason to go with an alpha version? Isn't the last GA doesn't have the required features?

Reply:
No. We can probably update this to the latest version.

Reply:
We used the [material UI Lab](https://material-ui.com/components/about-the-lab/) for partial star ratings. It has released alpha versions yet as listed in https://www.npmjs.com/package/@material-ui/lab.